tests/issue/issue.cpp

```diff
@@ -480,3 +579,21 @@ TEST(Issue, isHint)
     e->setLevel(libcellml::Issue::Level::HINT);
     EXPECT_EQ(e->level(), libcellml::Issue::Level::HINT);
 }
+
+TEST(Issue, url)
+{
+    auto expectedUrl = "https://cellml-specification.readthedocs.io/en/latest/reference/formal_and_informative/specB01.html?issue=2.1.1";
+
+    auto issue = libcellml::Issue::create();
+    issue->setReferenceRule(libcellml::Issue::ReferenceRule::MODEL_NAME);
+    EXPECT_EQ(expectedUrl, issue->url());
+}
+
+TEST(Issue, undefinedIssueUrl)
+{
+    std::string e;
+    e = "";
```

Comment:
Replace with:
```
const std::string e;
```

Reply:
I wanted to make it clear that the string was empty for the test.  It can't be initialised to an empty string in the declaration because clang tidy complains, hence the slightly awkward definition here.